pkg/pull/pull.go

```diff
@@ -600,16 +591,14 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		}
 	}
 
-	installationBytes, err := ioutil.ReadFile(filepath.Join(u.GetUpstreamDir(writeUpstreamOptions), "userdata", "installation.yaml"))
+	// installation spec gets updated during the process, ensure the map has the latest version
```

Comment:
at this point, would the `renderedKotsKinds.Installation` be the same as what's in `upstream/userdata/installation.yaml`? just curious why we need to reload it from the archive vs marshaling what we have in `renderedKotsKinds`

Reply:
they would be the same, yes. it didn't really matter which one to use. i decided to load it from the upstream directory just to make sure it's definitely the same, and just in case someone forgets to update the `renderedKotsKinds` object in a later change.